manual/source/utilities.rst

```diff
@@ -141,41 +141,36 @@ The list is not intended to be a complete list of all available packages.
 .. index::
    validation
    file; validate
-   nxvalidate (utility)
+   cnxvalidate (utility)
 
-.. _nxvalidate:
+.. _cnxvalidate:
 
-**nxvalidate**
-   From the source code documentation: 
+**cnxvalidate**
+   NeXus validation tool written in C (not via NAPI).
     
-   *"This is the first version of nxvalidate written in C. 
-   Its dependencies are libxml2 and the HDF5 libraries, 
-   version 1.8.9 or better. Its purpose is to validate HDF5 
-   data files against NeXus application definitions."*
-    
-   Note: this tool is lives in its own GitHub repository **cnxvalidate**:
+   Its dependencies are libxml2 and the HDF5 libraries, version 1.8.9 or
+   better. Its purpose is to validate HDF5 files against NeXus
+   application definitions. 
+
+   See the program documentation for more details:
    https://github.com/nexusformat/cnxvalidate.git
 
 .. index::
         validation
    file; validate
    punx (utility)
 
-.. _punx:
```

Comment:
Is hyperlink target unused?

Reply:
Adding those links where it was used in the h5py examples.  Moving the hyperlink target from validation.rst to here.

Reply:
2nd thought, leave the hyperlink target in validations